core/src/scene/sceneLoader.cpp

```diff
@@ -990,6 +992,24 @@ void SceneLoader::loadSource(const std::shared_ptr<Platform>& platform, const st
         url = urlStream.str();
     }
 
+    auto constructUrls = [&](int loc, const std::string& subD) {
+        urls.push_back(url);
+        urls.back().replace(loc, 3, subD);
+    };
+
+    if (Node subDomainNode = source["url_subdomains"]) {
+        auto loc = url.find("{s}");
+        if (loc != std::string::npos && subDomainNode.IsSequence()) {
+            for (const auto& domain : subDomainNode) {
+                if (domain.IsScalar()) {
+                    constructUrls(loc, domain.Scalar());
```

Comment:
Oh good, I was questioning my eyesight for a second there haha

True, subdomains aren't exactly the same as tile coordinates. To me it still seems simpler to substitute at the time of a request if we have a list of subdomains - otherwise we're doing more work for all the requests that won't use subdomains (which is most).